docs/usage.md

```diff
@@ -191,16 +196,17 @@ The mandatory and optional parameters for each category are tabulated below.
 
 #### 5. SNV annotation (ensembl VEP)
 
-| Mandatory                    | Optional                   |
-| ---------------------------- | -------------------------- |
-| vep_plugin_files<sup>1</sup> | skip_vep<sup>2</sup>       |
-|                              | vep_cache<sup>3</sup>      |
-|                              | vep_cache_version          |
-|                              | gene_panel_clinical_filter |
-
-<sup>1</sup> VEP caches can be downloaded [here](https://www.ensembl.org/info/docs/tools/vep/script/vep_cache.html#cache). VEP plugins may be installed in the cache directory, and the plugin pLI is mandatory to install. To supply files required by VEP plugins, use `vep_plugin_files` parameter. See example cache [here](https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/reference/vep_cache_and_plugins.tar.gz).<br />
-<sup>2</sup> If it is not provided by the user, the default value is false<br />
-<sup>3</sup> If it is not provided by the user, the default value is 110, supported values are 107 and 110 <br />
+| Mandatory | Optional                      |
+| --------- | ----------------------------- |
+|           | skip_vep<sup>1</sup>          |
+|           | vep_plugin_files<sup>2</sup>  |
+|           | vep_cache<sup>2</sup>         |
+|           | vep_cache_version<sup>3</sup> |
+|           | gene_panel_clinical_filter    |
+
+<sup>1</sup> If it is not provided by the user, the default value is false<br />
+<sup>2</sup> VEP cache and plugins can be automatically downloaded by the pipeline by setting `--skip_download_vep false`, `--skip_download_gnomad false` and providing a lcsv with a list of files to download `--vep_refs_download` as done [here](https://github.com/genomic-medicine-sweden/tomte/blob/dev/test_data/vep_to_download.csv). VEP caches can also be downloaded [here](https://www.ensembl.org/info/docs/tools/vep/script/vep_cache.html#cache). VEP plugins may also be installed in the cache directory, and the plugin pLI is mandatory to install. To supply files required by VEP plugins, use `vep_plugin_files` parameter. See example cache [here](https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/reference/vep_cache_and_plugins.tar.gz).<br />
```

Comment:
Can't find `--skip_download_vep`, `--skip_download_gnomad` and `--vep_refs_download` in a table, not sure if they should be or not.

Reply:
To be honest, I was doubting whether to add it or not, but it is likely better to have too much too much than too little 😄